### ISSUE TITLE
[css-2017] Fix the indices

### ISSUE THREAD

**Issue by frivoal** [w3c/csswg-drafts#935]
Update / create the index section of the 2017 snapshot

https://drafts.csswg.org/css-2017/#indices

@tabatkins Please either do this, or show me how to.

**Comment by frivoal** [comment]
@tabatkins I've tried to add the indices to the snapshot (https://github.com/frivoal/csswg-drafts/commit/7bdf5830e675169643e9b3f03bd1b320c0ff08e1) based on guessing what the syntax is supposed to be from skimming the source, but it seems I got it wrong.

```
<index type="dfn" status="TR" spec="
	css21,
	css-syntax-3,
	css-style-attr-1,
	mediaqueries-3,
	css-conditional-3,
        ....
"></index>
```
This does generate an index of the right kind, but neither the `status` not the `spec` filter seem to apply, and I get everyting bikeshed knows about. What's wrong?
